eng/common/scripts/Package-Properties.ps1

```diff
@@ -126,6 +126,14 @@ class PackageProps {
 
             if ($ciArtifactResult) {
                 $this.ArtifactDetails = [Hashtable]$ciArtifactResult.ArtifactConfig
+                $this.CIParameters["CITriggerPaths"] = @()
+                $this.CIParameters["CIMatrixConfigs"] = @()
+
+                $triggers = GetValueSafelyFrom-Yaml $ciArtifactResult.ParsedYml @("trigger", "paths", "include")
```

Comment:
If we don't plan on using these lets not even pull them in.

Reply:
Yep I just wanted to have it for discussion. I'll clean it up now.